server/controllers/medical/projects.js

```diff
@@ -22,8 +22,15 @@ exports.list = function list(req, res, next) {
   // send a larger response if complete is 1
   if (req.query.complete === 1) {
     sql = 'SELECT project.id, project.enterprise_id, project.abbr, ' +
-      'project.zs_id, project.name ' +
+      'project.zs_id, project.name, project.is_locked ' +
     'FROM project;';
+  } else if (req.query.unlocked === 1) {
+    sql = 'SELECT project.id, project.enterprise_id, project.abbr, ' +
+      'project.zs_id, project.name, project.is_locked ' +
+    'FROM project WHERE project.is_locked = \'0\' ;';
+  } else if (req.query.incomplete_unlocked === 1){
```

Comment:
It seems to me that this will be the usual query.  Why not have `GET /projects` only send back unlocked projects?  Then `GET /projects?locked=1` will send locked projects, `GET /projects?locked=1&complete=1